config/packages/security.yaml

```diff
@@ -36,13 +40,15 @@ security:
                 path: fos_user_security_logout
                 target: homepage
 
+
```

Comment:
empty line can be removed

Reply:
Fixed in 9944d94d01421fd5ea527806a2d3654727c45af8